src/figuresClasses.ts

```diff
@@ -1,19 +1,85 @@
 export interface Figure {
-
+  shape: string;
+  color: string;
```

Comment:
```suggestion
  shape: Shape;
  color: Color;
```
Consider creating a type to an enum for color and shape